spec/requests/cloud_databases_spec.rb

```diff
@@ -21,7 +23,7 @@
 
   describe 'cloud database actions' do
     let(:ems) do
-      FactoryBot.create(:ems_amazon)
+      FactoryBot.create(:ems_ibm_cloud_vpc)
```

Comment:
Does the specific type of ems matter here?  You're stubbing the params so I think we should be able to leave this as simply `:ems_cloud`?  If we can it is best to use generic types over specific provider types